backend/tests/integration/records/serializers/test_bibtex.py

```diff
@@ -139,3 +141,29 @@ def test_bibtex_search(api_client, db, es, create_record):
     assert expected_status_code == response_status_code
     assert expected_result_1 in response_data
     assert expected_result_2 in response_data
+
+
+def test_bibtex_with_math_environment_in_title(api_client, db, es, create_record):
```

Comment:
can you add tests for the corner case I mentioned, and a test checking that text is correctly encoded? These are strings you can use:
* `"Low-energy theorem for $\\gamma\\to 3\\pi$: Σ surface terms against $\\pi a_1$-mixing"` (this one is a bit fake, I added the `Σ`)
* `"Core polarization effects up to 12ℏω in 7Li and 10B nuclei"` (this one is real but latexcodec doesn't know about these characters, so it's not encoded properly, you should give the correct expected output and xfail the test).